scripts/install-crowdsec_openresty_bouncer

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -e
+
+BLUE='\E[1;34m'
+CYAN='\E[1;36m'
+YELLOW='\E[1;33m'
+GREEN='\E[1;32m'
+RESET='\E[0m'
+
+echo -e "${BLUE}❯ ${CYAN}Installing Crowdsec OpenResty Bouncer ${YELLOW}${CROWDSEC_OPENRESTY_BOUNCER_VERSION}...${RESET}"
```

Comment:
Use `${CROWDSEC_OPENRESTY_BOUNCER_VERSION:-}` instead of `${CROWDSEC_OPENRESTY_BOUNCER_VERSION}` so that the variable falls back to empty, it will be checked with my comment below